src/game_api/containers/custom_allocator.cpp

```diff
@@ -12,12 +12,12 @@ using CustomFreeFun = void*(void*, void*);
 void* custom_malloc(std::size_t size)
 {
     static CustomMallocFun* _malloc = (CustomMallocFun*)get_address("custom_malloc"sv);
-    static void* _alloc_base = OnHeapPointer<void>(*(size_t*)get_address("malloc_base"sv)).decode();
+    static void* _alloc_base = OnHeapPointer<void>(*(size_t*)get_address("malloc_base"sv)).decode(); // probably should be decode_local
```

Comment:
If i understand all about the custom allocator, it allocates the stuff in memory that is copied over when doing save state (?)

So then this definitely should be local, but maybe I'm wrong?

Reply:
Yes it should be local
https://github.com/spelunky-fyi/overlunky/pull/369/commits/f93ee54120bc70b1108a81a80c1c03b79c101724